frontend/src/scenes/project/Settings/CorrelationConfig.tsx

```diff
@@ -37,10 +36,7 @@ export function CorrelationConfig(): JSX.Element {
     return (
         <>
             <h2 className="subtitle" id="internal-users-filtering">
-                Correlation analysis exclusions{' '}
-                <LemonTag type="warning" className="uppercase ml-2">
-                    Beta
-                </LemonTag>
```

Comment:
Had a look through https://github.com/PostHog/posthog/labels/feature%2Fcorrelation-analysis and while most of the topics there are still open, I think this is mature enough to be out of "beta". We aren't getting feedback indicating otherwise as well.

Only thing that seems a bit broken right now is that adding "all events" turns up with empty correlations, since they all get filtered out by the threshold for when a correlation becomes relevant. Thinking of adding a warning message there.